curriculum/challenges/english/blocks/workshop-colorful-boxes/68ef0d6d156f86a6ac1824c6.md

```diff
@@ -0,0 +1,108 @@
+---
+id: 68ef0d6d156f86a6ac1824c6
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+Now we are going to use the `align-content` property. This property controls how rows of items are spaced out in your flex container. The most relevant options you can use in `align-content` flexbox are: `space-between`, `start`, `center`, `end`, `space-around` and `space-evenly`.
+
+Now change the `align-content` value to `space-between` to see how the boxes will align in the `flex-container`. By default the `align-content` value is `stretch`.
+
+
+# --hints--
+
+Your class `flex-container` should have a property `align-content` with the value `space-between`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.flex-container')?.getPropVal('align-content'), 'space-between');
+```
+
+# --seed--
+
+## --seed-contents--
+
+```html
+--fcc-editable-region--
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Colored Boxes</title>
+    
+  </head>
+  <body>
+    <header>
+      <h1>CSS flexbox colorful boxes</h1>
+    </header>
+    <div class="flex-container">
+      <div class="box">
+        <h3>Box 1</h3>
+        <p>Red</p>
+      </div>
+      <div class="box">
+        <h3>Box 2</h3>
+        <p>Orange</p>
+      </div>
+      <div class="box">
+        <h3>Box 3</h3>
+        <p>Yellow</p>
+      </div>
+      <div class="box">
+        <h3>Box 4</h3>
+        <p>Green</p>
+      </div>
+      <div class="box">
+        <h3>Box 5</h3>
+        <p>Blue</p>
+      </div>
+      <div class="box">
+        <h3>Box 6</h3>
+        <p>Indigo</p>
+      </div>
+    </div>
+  </body>
+</html>
+--fcc-editable-region--
```

Comment:
please make sure the editable region is in one single file, the set up you have here with `--fcc-editable-region--` in both files is invalid